app/docs/0.10.x/auth.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Authentication Reference
+---
+
+# Authentication Reference
+
+Client access to upstream API services is typically controlled by the application and configuration of 
+Kong authentication plugins. Authetication plugins are [documented individually][plugins]. 
+
+## Anonymous Access
+
+Prior to Kong 0.10.x, a given API could be configured to allow **only** authenticated access (by applying an
+[auth plugin][plugins]) or **only** anonymous access - it was not possible to have a given API allow some 
+users to be authenticated and others to access anonymously.
+
+Kong 0.10.x adds the ability to configure a given API to allow **both** authenticated **and** anonymous access. 
+You might use this configuration to grant access to anonymous users with a low rate-limit, and grant access 
+to authenticated users with a higher rate limit. 
+
+To configure an API like this, you first apply your selected authentication plugin, then create a new 
+consumer to represent annonymous users, then configure your authentication plugin to allow annonymous access. Here is an example, which assumes you have already configured an API named `example-api`:
+
+TODO: Try out the following steps and ensure they function as described
+
+1. ### Configure the key-auth plugin for your API
+
+    Issue the following cURL request on the API named `example-api`:
+
+    ```bash
+    $ curl -i -X POST \
+      --url http://localhost:8001/apis/example-api/plugins/ \
+      --data 'name=key-auth'
+    ```
+    
+2. ### Verify that the key-auth plugin is properly configured
+
+    Issue the following cURL request to verify that the [key-auth][key-auth]
```

Comment:
Perhaps it's better to move the documentation link to the first mention of `key-auth`; much like an acronym definition in an IEEE doc.